File: lib/models/user_role.dart
// lib/models/user_role.dart
enum UserRole {
  ATTENDEE,
  ORGANIZER,
  ADMIN,
}


UserRole userRoleFromString(String role) {
  return UserRole.values.firstWhere(
    (e) => e.name == role,
    orElse: () => UserRole.ATTENDEE,
  );
}

File: src/main/java/com/example/eventsphere/enums/UserRole.java
package com.example.eventsphere.enums;

import lombok.Getter;

@Getter
public enum UserRole {

    ATTENDEE("Attendee"),
    ORGANIZER("Organizer"),
    ADMIN("Admin");
    private final String label;

    UserRole(String label){
        this.label=label;
    }


    public String getLabel(){
        return label;
    }
}
